src/org/feedhenry/Utils.groovy

```diff
@@ -36,3 +38,9 @@ def getArtifactsDir(name) {
 def gitRepoIsDirty(untrackedFiles='no') {
     return sh(returnStdout: true, script: "git status --porcelain --untracked-files=${untrackedFiles}").trim()
 }
+
+static Closure getDate() {
```

Comment:
Yes, I think I got the return types wrong here, I'll change it. It's a pity it doesn't seem to check that :-/ I'll check also to see if the `@TypeChecked` annotation has any effect in pipelines.

I'll try static methods for the others, and make sure that they work even on an instance, so that we don't have to worry about changing existing call sites straight away (or at all)